src/index.html

```diff
@@ -3,12 +3,40 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo"><img src="images/logo.png" alt="Logo"></a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item "><a href="#apple" class="nav__link is-active">Apple</a></li>
+          <li class="nav__item"><a href="#samsung" class="nav__link">Samsung</a></li>
+          <li class="nav__item"><a href="#smartphones" class="nav__link">Smartphones</a></li>
+          <li class="nav__item">
+            <a href="#laptops-computers"
+              data-qa="hover"
+              class="nav__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item"><a href="#gadgets" class="nav__link">Gadgets</a></li>
+          <li class="nav__item"><a href="#tablets" class="nav__link">Tablets</a></li>
+          <li class="nav__item"><a href="#photo" class="nav__link">Photo</a></li>
+          <li class="nav__item"><a href="#video" class="nav__link">Video</a></li>
+        </ul>
+      </nav>
```

Comment:
Sorry my comment might have been missleading but in this case this formatting would be more readable
```html
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
          </li>
          <li class="nav__item">
            <a href="#samsung" class="nav__link">Samsung</a>
          </li>
          <li class="nav__item">
            <a href="#smartphones" class="nav__link">Smartphones</a>
          </li>
          <li class="nav__item">
            <a
              href="#laptops&computers"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
          </li>
          <li class="nav__item">
            <a href="#gadgets" class="nav__link">Gadgets</a>
          </li>
          <li class="nav__item">
            <a href="#tablets" class="nav__link">Tablets</a>
          </li>
          <li class="nav__item">
            <a href="#photo" class="nav__link">Photo</a>
          </li>
          <li class="nav__item">
            <a href="#video" class="nav__link">Video</a>
          </li>
        </ul>
      </nav>
```